composer.json

```diff
@@ -1,29 +1,31 @@
 {
-    "name":"codeception/module-mezzio",
-    "description":"Codeception module for Mezzio framework",
-    "keywords":["codeception", "mezzio"],
-    "homepage":"http://codeception.com/",
-    "type":"library",
-    "license":"MIT",
-    "authors":[
-        {
-            "name":"Gintautas Miselis"
-        }
-    ],
-    "minimum-stability": "RC",
-    "require": {
-        "php": ">=5.6.0 <9.0",
-        "codeception/lib-innerbrowser": "^1.0",
-        "codeception/codeception": "^4.0"
-    },
-    "require-dev": {
-        "codeception/module-rest": "dev-master",
-        "mezzio/mezzio": "~2.0 | ~3.0"
-    },
-    "autoload":{
-        "classmap": ["src/"]
-    },
-    "config": {
-        "classmap-authoritative": true
-    }
+	"name": "codeception/module-mezzio",
+	"description": "Codeception module for Mezzio framework",
+	"keywords": [ "codeception", "mezzio" ],
+	"homepage": "https://codeception.com/",
+	"type": "library",
+	"license": "MIT",
+	"authors": [
+		{
+			"name": "Gintautas Miselis"
+		}
+	],
+	"minimum-stability": "RC",
+	"require": {
+		"php": "^7.1 || ^8.0",
```

Comment:
Thank for taking this chance: since we require `mezzio:v3` we can bump this even further to match it (https://github.com/mezzio/mezzio/blob/f2643cac71621ff7dcf56091214f6cd14deaaf6c/composer.json#L33), so `"php": "^7.3 || ^8.0"`

Reply:
That is even better!

I did not do it initially because [I promised](https://github.com/Codeception/Codeception/issues/6164#issuecomment-822055749) to update too many modules and an update from PHP 5.6 to PHP 7.3 takes a long time. Even initially I just thought about adding return types...

But I will gladly apply the improvements of PHP 7.2 and PHP 7.3 since you asked for it :-)

Reply:
Even better :-)
I didn't do it right away because [I promised](https://github.com/Codeception/Codeception/issues/6164#issuecomment-822055749) to update a lot of modules, and in some cases going from PHP 5.6 to PHP 7.3 requires a lot of effort and time.